Lib/importlib/_bootstrap_external.py

```diff
@@ -1147,12 +1150,15 @@ def append(self, item):
         self._path.append(item)
 
 
+_NamespacePath = NamespacePath  # for backwards compatibility
```

Comment:
Should we make it a deprecated alias? (and emit a warning if we were to access the private name? that could help transition from the private to the public name).